docs/02-guides/flash-messages.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Flash messages
+description:
+  Learn how to push and display flash messages in your Front-Commerce
+  application.
+---
+
+<SinceVersion tag="3.6" />
+
+<p>{frontMatter.description}</p>
+
+## When to use flash messages?
+
+Flash messages are used to display informations to the user. These messages are
+meant to be displayed once and are stored in the session. When consummed, they
+are removed from the session.
+
+## Flash message data type
+
+Flash messages are represented by an object the following properties:
+
+| Property  | Type   | Description                                                                                                                    |
+| --------- | ------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| `type`    | string | The type of the flash message. It can be one of the types declared in the [flash messages component](#flash-messages-component). |
+| `message` | string | The message to display.                                                                                                        |
+| `data`    | object | An object that can be used to pass additional data to the flash message.                                                       |
+
+## Flash messages component
+
+Theme chocolatine provides default flash messages component depending on the
```

Comment:
**nitpick:** if that was a Theme Chocolatine specific things, it should go under it's own Extension section.\
In this case, it's more a default FC behavior than a specific theme one IMO.

```suggestion
The default theme provides a default implementation for flash messages depending on the
```

Reply:
fixed with your suggestion